.gitignore

```diff
@@ -4,3 +4,6 @@ Cargo.lock
 #IntelliJ project files
 .idea
 *.iml
+
+CMakeLists.txt
+cmake-build-debug
```

Comment:
Do you need these?

Reply:
They help me because this is what makes CLion work good but I can remove that if you don't want it in the gitignore

Reply:
No worries, I don't think they hurt us, I was just wondering if this was a mistake.